draft-ietf-teep-protocol.md

```diff
@@ -438,6 +441,42 @@ versions
   If this field is not present, it is to be treated the same as if
   it contained only version 0.
 
+attestation-payload-format
```

Comment:
I don't see why these are needed in the QueryResponse, they should be in an Error message instead.

Reply:
Moved them to in the Error message.